README.md

```diff
@@ -2,7 +2,7 @@
 
 ### Project Status
 
-[![Coverage Status](https://coveralls.io/repos/github/CNERG/trailmap/badge.svg?branch=main)](https://coveralls.io/github/CNERG/trailmap?branch=main) 
+[![Coverage Status](https://coveralls.io/repos/github/bquan0/trailmap/badge.svg?branch=github-actions)](https://coveralls.io/github/bquan0/trailmap?branch=github-actions)
```

Comment:
This should stay as it was originally.

It may have been useful for testing your branch, but it will refer to the wrong data once merged.